6. Developer Guides/09. Realtime-API/1. Method Calls/6. Get Subscriptions/README.md

```diff
@@ -0,0 +1,12 @@
+# Subscriptions Get
+
+TOOD: I have no returned data here. Need back up.
```

Comment:
I got this incredible idea of logout and login again. I got the return data. Working on that.